src/azure-cli-core/azure/cli/core/cloud.py

```diff
@@ -334,18 +402,58 @@ def __str__(self):
         postgresql_server_endpoint='.postgres.database.cloudapi.de',
         mariadb_server_endpoint='.mariadb.database.cloudapi.de'))
 
-KNOWN_CLOUDS = [AZURE_PUBLIC_CLOUD, AZURE_CHINA_CLOUD, AZURE_US_GOV_CLOUD, AZURE_GERMAN_CLOUD]
+KNOWN_CLOUDS = []
```

Comment:
I think it is better to use delegate function here like KNOWN_CLOUDS = get_known_clouds(), the logic looks complicated to update `global KNOWN_CLOUDS` in get_know_clouds while you still have a public variable KNOWN_CLOUDS, it is easy to make mistake